Intersect.Editor/Localization/Strings.cs

```diff
@@ -3815,7 +3815,7 @@ public partial struct MainForm
             public static LocalizedString file = @"File";
 
             [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
-            public static LocalizedString Fill = @"Fill Layer Tool (F)";
+            public static LocalizedString Fill = @"Layer Fill Tool (F)";
```

Comment:
I don't think this string change makes sense, all of the tools (except the Eyedropper which conflicts with Eraser -- and Eyedropper has a more natural alternative with `i`) have a keybind that is the first letter of their tool name (at least in English).

That said, I do support changing all of the tool strings -- specifically drop the word "Tool" from the tool strings